RGPosadas/LitterRally
# US-7.6: Login as a User

This PR closes #99.

**Steps to test**
1. Launch the application using `nx serve mull-ui` & `nx serve mull-api`.
2. Navigating to any route except `/login` & `/register` should redirect you to `/login`
3. Create an account if you do not have one to test with.
4. Login with email and pass.
5. In the application tab of the browser developer tools, you should see a cookie called `mullToken`.
6. Closes the browser or tab.
7. Go back to the application.
8. You should be logged in as the same user.
9. Clear the cookies of the browser.
10. Repeat steps 5 to 8 for OAuth login.
